apps/condo/domains/notification/schema/SendMessageService.js

```diff
@@ -72,12 +72,12 @@ const SendMessageService = new GQLCustomSchema('SendMessageService', {
             resolver: async (parent, args, context, info, extra = {}) => {
                 // TODO(pahaz): think about sending emails with attachments
                 const { data } = args
-                const { dv, sender, to, type, meta, lang } = data
+                const { dv, sender, to, emailFrom, type, meta, lang } = data
```

Comment:
I think you need to check that you can use `emailFrom` only with `to.email` argument. It is useless in other cases.

Reply:
Done